services/venice-router/src/test/java/com/linkedin/venice/router/throttle/RouterRequestThrottlingTest.java

```diff
@@ -72,7 +74,12 @@ public static Object[][] requestType() {
     return new Object[][] { { RequestType.MULTI_GET }, { RequestType.COMPUTE } };
   }
 
-  @Test(timeOut = 30000, dataProvider = "multiGet_compute")
+  /** A simple Runnable-like interface just to reduce boilerplate in the test... */
+  private interface ScatterCall {
+    void run() throws RouterException;
+  }
+
+  @Test(timeOut = 30000, dataProvider = "multiGet_compute", invocationCount = 100)
```

Comment:
Is the `invocationCount = 100` intentional?

Reply:
Oh, no it isn't, thanks for catching that. I only did this to make sure my de-flaking worked.

Reply:
This is now addressed in the latest commit, but needs another ship it...